### PR TITLE
user model added

### PR DESCRIPTION
**Pull request by AnasSabbag127**

### CODE REVIEW

File: src/model/post_info.rs
Comment (Atul-Bhatt-RS) [inline src/model/post_info.rs:1]: Change Model name from PostInfo to Post and post_info field to post_text or text.

File: src/model/comment.rs
Comment (Atul-Bhatt-RS) [inline src/model/comment.rs:1]: Post Id is required.

File: src/model/post_info.rs
Comment (Atul-Bhatt-RS) [inline src/model/post_info.rs:1]: User ID is required,

**Review by Atul-Bhatt-RS — CHANGES_REQUESTED** [review]

File: src/model/post_info.rs
Comment (AnasSabbag127) [reply src/model/post_info.rs:1]: i made the neccessary changes

Comment (AnasSabbag127) [reply src/model/post_info.rs:1]: done

File: src/model/comment.rs
Comment (AnasSabbag127) [reply src/model/comment.rs:1]: done

**Review by Atul-Bhatt-RS — APPROVED** [review]

**Pull request merged**